CHANGELOG.md

```diff
@@ -93,6 +98,15 @@ Additions to existing modules
   _≡?_ : DecidableEquality (Vec A n)
   ```
 
+* In `Function.Related.TypeIsomorphisms`:
+  ```agda
+  Σ-distribˡ-⊎ : {P : A → Set a} {Q : A → Set b} → (∃ λ a → P a ⊎ Q a) ↔ (∃ P ⊎ ∃ Q)
+  Σ-distribʳ-⊎ : {P : Set a} {Q : Set b} {R : P ⊎ Q → Set c} → (Σ (P ⊎ Q) R) ↔ (Σ P (R ∘ inj₁) ⊎ Σ Q (R ∘ inj₂))
+  ×-distribˡ-⊎′ : (A × (B ⊎ C)) ↔ (A × B ⊎ A × C)
+  ×-distribʳ-⊎′ : ((A ⊎ B) × C) ↔ (A × C ⊎ B × C)
+  ∃-≡ : ∀ (P : A → Set b) {x} → P x ↔ (∃[ y ] y ≡ x × P y)
```

Comment:
Is there also a place for the *right* Kan extension version ('Yoneda') of this lemma (characterising the *left* Kan extension of `P` along `_≡_`, cf. 'coYoneda'), viz:
```agda
∀-≡ : ∀ (P : A → Set b) {x} → P x ↔ (∀ {y} → y ≡ x → P y)
```

Reply:
Oh... the R-to-L inverse map seems to need `Extensionality` and/or (?) `with-K`? But L-to-R works fine... so can the statement be suitably weakened (I'm not up-to-speed with this bit of the library and all the `*Inverse*` definitions...

Reply:
Postpone to a downstream PR!